packages/plugins/documentation/admin/src/pages/PluginPage/usePluginPage.js

```diff
@@ -0,0 +1,56 @@
+import { useQuery, useMutation, useQueryClient } from 'react-query';
+import { useNotification } from '@strapi/helper-plugin';
+import { fetchData, deleteDoc, regenerateDoc, submit } from './utils/api';
```

Comment:
Could you please rename "submit" on something a bit more meaningful please? 🙏🏻 😊